opendbc/car/hyundai/values.py

```diff
@@ -539,6 +539,10 @@ class CAR(Platforms):
     CarSpecs(mass=2060, wheelbase=3.01, steerRatio=16.5),
     flags=HyundaiFlags.LEGACY,
   )
+  GENESIS_G80_2ND_GEN_FL = HyundaiCanFDPlatformConfig(
+    [HyundaiCarDocs("Genesis G80 (2.5T Advanced Trim, with HDA II) 2024", "Highway Driving Assist II", car_parts=CarParts.common([CarHarness.hyundai_p]))],
+    CarSpecs(mass=2060, wheelbase=3.00, steerRatio=11.2),
```

Comment:
![image](https://github.com/user-attachments/assets/e0cf1c60-1476-4583-86bc-7a0e60beb7c8)

```suggestion
    CarSpecs(mass=2060, wheelbase=3.00, steerRatio=14.0),
```